### ISSUE TITLE
[bug] Fetch xml to OData v4 adds `...&utm_source=FetchXMLBuilder&utm_medium=XrmToolBox`

### ISSUE THREAD

**Issue by rafek1241** [rappen/FetchXMLBuilder#437]
Fetch xml to OData v4 adds `...&utm_source=FetchXMLBuilder&utm_medium=XrmToolBox` at the end of the url.

When i'm clicking the url generated for specific fetch xml query it append mentioned get parameters.
In a results D365 Dataverse returns this message:
```
{"error":{"code":"0x0","message":"The query parameter utm_source is not supported"}}

```

**Comment by rappen** [comment]
Ouch! 🤦‍♂️
Thanks for reporting, update will be published later today.

**Comment by rappen** [comment]
Fix is now released in version 1.2020.12.3.
